postBuild

```diff
@@ -1 +1,5 @@
-pip install .[tutorials]
\ No newline at end of file
+pip install .[tutorials]
+
+# Install conda version of mpi4py for mybinder service
+conda install mpi4py
```

Comment:
Does this imply that the tests are never run with the pip installed version of mpi4py?

Reply:
pip install mpi4py requires libmpi linux package to be installed. This is the workaround for both readthedocs and mybinder.